src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,150 @@
+
+    ```java
+    //Alpha modules
+    implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+    ```
+
+    </Step>
+    <Step>
+
+### Add OpenTelemetry API and SDK dependencies
+        
+In the `dependencies()` argument, add the OpenTelemetry API and SDK:
+
+    ```java
+    dependencies {
+        implementation 'io.opentelemetry:opentelemetry-api'
+        implementation 'io.opentelemetry:opentelemetry-sdk'
+    }
+    ```
+    
+An example completed file might look something like this:
+
+    ```java
+    plugins {
+        id 'java-library'
+        id 'org.springframework.boot'
+    }
+
+    bootRun {
+        mainClass.set 'io.opentelemetry.example.micrometer.Application'
+    }
+
+    dependencies {
+        //OpenTelemetry API and SDK
+        implementation 'io.opentelemetry:opentelemetry-api'
+        implementation 'io.opentelemetry:opentelemetry-sdk'
+        //Alpha modules where Micrometer instrumentation appears
+        implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+        implementation 'io.opentelemetry:opentelemetry-exporter-otlp'
+
+        implementation platform(SpringBootPlugin.BOM_COORDINATES)
+        implementation 'org.springframework.boot:spring-boot-starter-web'
+        implementation 'org.springframework.boot:spring-boot-starter-actuator'
+        implementation 'org.springframework.boot:spring-boot-starter-aop'
+    }
+    ```
+
+    </Step>
+    <Step>
+
+### Configure Micrometer to forward data to New Relic
+
+Below is an example code snippet that directs Micrometer to use the OpenTelemetry Micrometer bridge. This snippet updates your code so OpenTelemetry can:
+
+* Detect Micrometer data
+* Forward that data to New Relic
+
+```java
+public OpenTelemetry openTelemetry() {
+    return OpenTelemetrySdk.builder()
+        .setMeterProvider(
+            SdkMeterProvider.builder()
+                .setResource(
+                    Resource.getDefault().toBuilder()
+                        .put("service.name", "micrometer-shim")
+                        // Include instrumentation.provider=micrometer to enable micrometer metrics
+                        // experience in New Relic
+                        .put("instrumentation.provider", "micrometer")
+                        .build())
+                .registerMetricReader(
+                    PeriodicMetricReader.builder(
+                            OtlpHttpMetricExporter.builder()
+                                .setEndpoint("https://otlp.nr-data.net")
+                                .addHeader(
+                                    "api-key",
+                                    Optional.ofNullable(System.getenv("NEW_RELIC_LICENSE_KEY"))
+                                        .filter(str -> !str.isEmpty() && !str.isBlank())
+                                        .orElseThrow())
+                                // IMPORTANT: New Relic requires metrics to be delta temporality
```

Comment:
What is delta temporality here? What exactly is neW reLIC requiring?

Reply:
Aggregation temporality determines whether metrics which are reported represent the cumulative or the delta since the last report. For example, if you have a counter which is incremented every minute and you report / export it every minute, cumulative temporality would export values `1, 2, 3, 4, ..`, while delta would export values `1, 1, 1, 1, ...`. 

New Relic is a delta metrics backend. We support cumulative, but for various reasons its better for everyone to send us delta.

Reply:
woah! TIL! Super cool tidbit, thanks!